docs/registry/getting-started-registry/README.adoc

```diff
@@ -201,32 +205,34 @@ NOTE: Specifying a group is optional when using the web console, and a `default`
   ]
 }
 ----
+** *From URL*: Enter a valid and accessible HTTPS URL, and click *Fetch*. For example: `\https://petstore3.swagger.io/api/v3/openapi.json`.
 
 . Click *Upload* to complete the operation and display the new artifact details:
 
 * *Overview* tab:
 ** *Version metadata*: Displays details such as the artifact name, artifact ID, global ID, content ID, labels, properties, and so on.
-** *Content rules*: Displays artifact content rules that you can enable and configure. You can configure a *Validity rule* or *Compatibility rule* by enabling the rule and then selecting the appropriate rule configuration from the list. For details on supported rules, see the https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry/1[{product-long-registry} user documentation^].
+** *Content rules*: Displays artifact content rules that you can enable and configure. You can configure a *Validity rule* or *Compatibility rule* by enabling the rule and then selecting the appropriate rule configuration from the list. For details on supported rules, see {content-rules-registry}.
+
 * *Documentation* tab: (OpenAPI only): Displays automatically-generated REST API documentation.
```

Comment:
```suggestion
* *Documentation* tab: (AsyncAPI or OpenAPI only): Displays automatically-generated REST API documentation.
```

Maybe check with Dev in case it applies to other types too?